README.md

```diff
@@ -38,6 +38,7 @@ Elemental2 is split up in several jars files. Refer to the following targets in
  promise | `@com_google_elemental2//java/elemental2/promise`
```

Comment:
I think we should not add webassembly here yet because RC1 doesn't include it.  You don't have to do anything, I'll revert that file during the import